implementations/generate.js

```diff
@@ -43,7 +43,12 @@ const deprecatedTests = [
   'Linked Data Proofs (optional) Linked Data Signature proofPurpose MUST exist and be "credentialIssuance"',
   'Terms of Use (optional) MUST support prohibiting Archival',
   'Terms of Use (optional) MUST support prohibiting non-subject Presentation',
-  'Terms of Use (optional) MUST support prohibiting 3rd Party Correlation'
+  'Terms of Use (optional) MUST support prohibiting 3rd Party Correlation',
+  'Basic Documents Presentations MUST include `verifiableCredential` and `proof`',
+  'Basic Documents Presentations MUST include `verifiableCredential` and `proof` (negative - missing `verifiableCredential`)',
+  'Basic Documents Presentations MUST include `verifiableCredential` and `proof` (negative - missing `proof`)',
```

Comment:
a presentation without a proof is valid according to the spec, this language does not feel precise enough for me.

Reply:
This is the list of deprecated tests, to which this PR adds 5. This list is used to remove the named tests from showing up in the generated `implementations/index.html`, even though they still may be present in some of the JSON reports.
The rows for the 5 deprecated tests are also removed from `implementations/index.html` in this PR.